EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+deportealdia.live,techgeek.digital###overlay
```

Comment:
This rule is for what?

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/74120#issuecomment-774494139

Reply:
Ok, but I can't reproduce it. @Yuki2718 can you provide a screenshot for this element?